.gitlab/e2e.yml

```diff
@@ -36,9 +36,9 @@ k8s-e2e-main:
   extends: .k8s_e2e_template
   allow_failure: true # temporary while investigating
   rules: !reference [.on_main]
-  # needs:
-  #   - dev_master-a6
-  #   - dev_master-a7
+  needs:
+    - dev_master-a6
```

Comment:
I'm wondering if we should remove a6 since we only deploye agent 7 internally.

Also I think we also need to way cluster-agent build

Reply:
> I'm wondering if we should remove a6 since we only deploye agent 7 internally.

If we want to do that, then I think we would want to remove line 43 from this file, which triggers the e2e script with `--agent-image=datadog/agent-dev:master-py2`

Personally I am fine with it, but I don't know if we want to keep the visibility for this package in place, especially since we don't actually deploy it anywhere else anymore. Thoughts?